docs/configuration/access-control.md

```diff
@@ -70,6 +70,11 @@ For a user with unique identifier `john`, the subject should be `user:john` and
 uniquely identified by `developers`, the subject should be `group:developers`. Similar to resources
 and domains you can define multiple subjects in a single rule.
 
+If you want a combination of subjects to be matched at once, you can specify a list of subjects like
+`- [group:developers, group:admins]`. Make sure to preceed it by a list key `-`.
```

Comment:
```suggestion
`- ["group:developers", "group:admins"]`. Make sure to preceed it by a list key `-`.
```

Reply:
Thanks, must have lost focus on the docs - sorry.